AppCheckCore/Sources/DebugProvider/GACAppCheckDebugProvider.m

```diff
@@ -132,6 +117,70 @@ - (void)getTokenWithLimitedUse:(BOOL)limitedUse
       });
 }
 
+static NSString *LocalDebugToken() {
+  return StoredDebugToken() ?: GenerateAndStoreDebugToken();
+}
```

Comment:
Thanks for catching this, Nick! You're right that the intention was for `GenerateAndStoreDebugToken` to be nonnull (since `[NSUUID UUID].UUIDString` will never produce `nil`).